### PR TITLE
#3030 Introduce the basic benchmark flow

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3030)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Topic.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+package com.island.ohara.it.benchmark
+
+import com.island.ohara.common.setting.TopicKey
+import com.island.ohara.common.util.CommonUtils
+import com.island.ohara.it.category.BenchmarkGroup
+import org.junit.Test
+import org.junit.experimental.categories.Category
+
+@Category(Array(classOf[BenchmarkGroup]))
+class Benchmark4Topic extends BasicBenchmark {
```
Comment (jackyoh) [inline ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Topic.scala]: 這邊很像要使用 abstract class Benchmark4Topic, 因為如果是測試 class 的話應該是 TestBenchmark4Topic

File: ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Perf.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import com.island.ohara.common.setting.{ConnectorKey, TopicKey}
+import com.island.ohara.common.util.CommonUtils
+import com.island.ohara.connector.perf.PerfSource
+import com.island.ohara.it.category.BenchmarkGroup
+import org.junit.Test
+import org.junit.experimental.categories.Category
+
+@Category(Array(classOf[BenchmarkGroup]))
+class Benchmark4Perf extends BasicBenchmark {
```
Comment (jackyoh) [inline ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Perf.scala]: ditto

File: ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Topic.scala (hunk 1 above)
Comment (chia7712) [reply ohara-it/src/test/scala/com/island/ohara/it/benchmark/Benchmark4Topic.scala]: nice finding. let me fix it :)

**Comment by chia7712** [comment]
error is unrelated and traced by #3428

**Pull request merged**